### PR TITLE
Update README.md to improve Installation readability

### PR DESCRIPTION
**Pull request by crushingCodes**
- fix the missing pair of ``` making it harder to find the installation instructions in Readme.md
- added some markdown headings for the relevant installation method to make it easier to find relevant info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,15 +38,19 @@ You may then:
 - `-a` or `--api-key` will store your API key in the local keychain
 - `-v` or `--version` will show the current version
 - `-h` or `--help` will show the help message
+```
```
Comment (thomas-endres-tng) [inline README.md:41]: Why is this line necessary?

**Comment by thomas-endres-tng** [comment]
Could you please sign off your commit (using the Git -s option)?

File: README.md (hunk 1 above)
Comment (crushingCodes) [reply README.md:41]: @thomas-endres-tng Thanks for reviewing. This is to allow markdown to show Installation as a heading, i think it was missed at some point.

Screenshot of the current Readme.md:
Screenshot 2023-05-09 at 10 23 20 am

**Comment by crushingCodes** [comment]
> Could you please sign off your commit (using the Git -s option)?

I will do, just need to setup signing on my local machine
